src/components/pages/profile/MesInformations/__specs__/MesInformations.spec.jsx

```diff
@@ -1,105 +1,31 @@
-          getFormValuesByNames={jest.fn()}
-          handleSubmit={jest.fn()}
-          user={state.user}
-        />
-      </Router>
-    )
-
-    // Then
-    const publicNameLabel = wrapper.find({ children: 'Identifiant' })
-    const nameLabel = wrapper.find({ children: 'Nom et prénom' })
-    const emailLabel = wrapper.find({ children: 'Adresse e-mail' })
-    const departmentCodeLabel = wrapper.find({ children: 'Département de résidence' })
-    const passwordLabel = wrapper.find({ children: 'Mot de passe' })
-    expect(publicNameLabel).toHaveLength(1)
-    expect(nameLabel).toHaveLength(1)
-    expect(emailLabel).toHaveLength(1)
-    expect(departmentCodeLabel).toHaveLength(1)
-    expect(passwordLabel).toHaveLength(1)
-  })
-
-  it("should display beneficiary's profile informations", () => {
-    // Given
-    const user = { ...state.user }
-    user.publicName = 'Martino'
-    user.firstName = 'Martin'
-    user.lastName = 'Dupont'
-    user.email = 'm.dupont@test.com'
-    user.departementCode = '93'
-    const mockedGetDepartment = jest.fn().mockReturnValue('Seine-Saint-Denis (93)')
-
-    // When
-    const wrapper = mount(
-      <Router history={createBrowserHistory()}>
-        <MesInformations
-          getDepartment={mockedGetDepartment}
-          getFormValuesByNames={jest.fn()}
-          handleSubmit={jest.fn()}
-          user={user}
-        />
-      </Router>
+  it('should display a link to personal informations page', () => {
+    // given
+    const wrapper = shallow(
+      <MesInformations
+        getDepartment={jest.fn()}
+        getFormValuesByNames={jest.fn()}
+        handleSubmit={jest.fn()}
+        user={state.user}
+      />
     )
 
-    // Then
-    const publicName = wrapper.find("input[value='Martino']")
-    const name = wrapper.find("input[value='Martin Dupont']")
-    const email = wrapper.find("input[value='m.dupont@test.com']")
-    const departmentCode = wrapper.find("input[value='Seine-Saint-Denis (93)']")
-    expect(publicName).toHaveLength(1)
-    expect(name).toHaveLength(1)
-    expect(email).toHaveLength(1)
-    expect(departmentCode).toHaveLength(1)
-  })
-
-  it('should prevent name, email and departmentCode modifications', () => {
-    // Given
-    const user = { ...state.user }
-    user.publicName = 'Martino'
-    user.firstName = 'Martin'
-    user.lastName = 'Dupont'
-    user.email = 'm.dupont@test.com'
-    user.departementCode = '93'
-    const mockedGetDepartment = jest.fn().mockReturnValue('Seine-Saint-Denis (93)')
-
-    // When
-    const wrapper = mount(
-      <Router history={createBrowserHistory()}>
-        <MesInformations
-          getDepartment={mockedGetDepartment}
-          getFormValuesByNames={jest.fn()}
-          handleSubmit={jest.fn()}
-          user={user}
-        />
-      </Router>
-    )
+    // when
+    const linkToPersonalInformationsPage = wrapper
+      .find({ children: 'Informations personnelles' })
+      .parent()
```

Comment:
```suggestion
    const linkToPersonalInformationsPage = wrapper
      .find({ children: 'Informations personnelles' })
      .parent().props().to
```
A mettre dans ton `then`.